_field-types/mapping-parameters/copy-to.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: copy_to
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 20
+has_children: false
+has_toc: false
+---
+
+# `copy-to`
+
+The `copy-to` parameter allows you to copy the values of multiple fields into a single field. This can be useful if you often search across multiple fields, as it allows you to search the group field instead. 
```

Comment:
```suggestion
The `copy_to` parameter allows you to copy the values of multiple fields into a single field. This can be useful if you often search across multiple fields, as it allows you to search the group field instead. 
```